application/src/main/java/org/openhds/mobile/provider/DatabaseAdapter.java

```diff
@@ -57,6 +76,13 @@
 			+ SUPERVISOR_TABLE_NAME + " (" + KEY_ID + " INTEGER PRIMARY KEY, "
 			+ KEY_SUPERVISOR_NAME + " TEXT, " + KEY_SUPERVISOR_PASS + " TEXT)";
 
+	private static final String RECENT_DB_FORMS = "CREATE TABLE "
```

Comment:
While I don't particularly like the naming convention, we should use them to be consistent - or change them all to be consistent.